pkg/provisioners/kafka/controller/reconcile.go

```diff
@@ -42,6 +42,12 @@ const (
 func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	ctx := context.TODO()
 	r.logger.Info("reconciling ClusterChannelProvisioner", zap.Any("request", request))
+
+	// The reconcile requests triggered because of objects owned by this ClusterChannelProvisioner (e.g k8s service)
+	// will contain the namespace of that object. Since ClusterChannelProvisioner is cluster-scoped we need to unset the
+	// namespace or otherwise the provisioner object cannot be found.
```

Comment:
Can you add a reference to https://github.com/kubernetes-sigs/controller-runtime/issues/214 in this comment?

Reply:
Done